src/Secure/Login.php

```diff
@@ -64,23 +87,43 @@ class Login
      */
     private $cluster = 'https://c3.twinfield.com';
 
+    private $office;
+    private $organisation;
+
     public function __construct(Config $config)
     {
         $this->config = $config;
+        $this->office = $config->getOffice();
+        $this->organisation = $config->getOrganisation();
         $this->cluster = !is_null($config->cluster) ? $config->cluster : $this->cluster;
-        $this->loginService = new LoginService(null, $config->getSoapClientOptions());
+
+        $this->loginService = new LoginService(
+            null,
+            $config->getSoapClientOptions()
+        );
     }
 
     /**
      * @throws Exception
      */
     protected function login()
```

Comment:
deze functie heeft alleen in de if statement een return waarde. Klopt dat? De overige situaties hoeft niks teruggegeven worden?

Reply:
Ja dat klopt.